### DISCUSSION TITLE
Stitching types on mutation responses

### DISCUSSION THREAD

**Discussion by rocketraman** [hasura/graphql-engine#7142] [Question] [answered]
I have a remote schema. Is there a way to "stitch" return fields of the remote schema to Hasura entities via id's? I want the client to be able to perform mutations via the remote schema, with the ability to query Hasura entities as part of the mutation response.

**Comment by coco98** [comment, marked as answer]
@rocketraman This is immediately a part of the work on the new remote joins roadmap. We're expecting this to land in July!
Here's the issue you can subscribe to: https://github.com/hasura/graphql-engine/issues/5801

We planned to do this first, before we did database to database remote joins, but realized that the foundation settings required database to database joins first.

**Reply by rocketraman** [reply]
Awesome! Looking forward to the feature and have subscribed to #5801 .
